src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/ProteinsList.scss

```diff
@@ -1,4 +1,13 @@
 .proteinsList {
   font-size: 13px;
   font-weight: 300;
+  overflow: auto;
+}
+
+.proteinsListLoadingContainer {
+  height: 100%;
```

Comment:
Given this:

![image](https://user-images.githubusercontent.com/6834224/95102025-c6303100-072a-11eb-94e5-ce62892e77ee.png)

what's the purpose of height: 100% and flex-direction: column?

Reply:
I copied it from here:
https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.scss#L5

I have removed height and flex-direction as they don't make any difference.